.gitignore

```diff
@@ -78,8 +78,8 @@ playground.xcworkspace
 # you should judge for yourself, the pros and cons are mentioned at:
 # https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
 #
-# Pods/
-#
+ Pods/
```

Comment:
It doesn't appear to have ignored the Pod folder. Please use the following command to reset all tracked files and add them back on: `git rm -r --cached .` then, `git add .` Add a new commit `git commit -m "Remove Tracked Files"`

Reply:
see [this commit](https://github.com/MakeSchool-Companion-App/Companion-iOS/pull/2/commits/ec203fe69426e333a7ed539ffbc2a27899a400fb) for details on fix.